Resources/Prototypes/Entities/Objects/Tools/tools.yml

```diff
@@ -497,3 +497,27 @@
       Wood: 50
   - type: StaticPrice
     price: 25
+
+- type: entity
+  parent: BaseItem
+  id: RollingPin
+  name: rolling pin
+  description: Used to knock out the Bartender.
```

Comment:
Come up with a better description for this item please.

Reply:
i have no creativity

Reply:
"A tool used to shape and flatten dough."